modules/ROOT/pages/demo/full-featured.adoc

```diff
@@ -17,13 +19,14 @@ The following plugins are excluded from this example:
 
 * xref:plugins/premium/moxiemanager.adoc[MoxieManager (Premium Plugin)] -- xref:plugins/premium/tinydrive.adoc[*{cloudfilemanager}*] included instead.
 * xref:plugins/opensource/autoresize.adoc[Autoresize] -- Resizes the editor to fit the content.
-* xref:plugins/opensource/bbcode.adoc[BBCode] -- Changes the markup used for the content.
+* xref:plugins/opensource/bbcode.adoc[BBCode] -- Changes the markup used for the content. xref:release-notes/6.0-upcoming-changes.adoc#plugins[Deprecated] as of {productname} 5.9.
 * xref:plugins/opensource/code.adoc[Code] -- xref:plugins/premium/advcode.adoc[*Advanced Code Editor*] included instead.
-* xref:plugins/opensource/fullpage.adoc[Full Page] -- Used for modifying HTML `<head>` elements.
+* xref:plugins/opensource/fullpage.adoc[Full Page] -- Used for modifying HTML `<head>` elements. xref:release-notes/6.0-upcoming-changes/#plugins[Deprecated] as of {productname} 5.9.
```

Comment:
All these links still need to be fixed as per @MitchCTiny's previous comment.

Reply:
All these links now fixed as per @MitchCTiny’s previous comment.